rq/job.py

```diff
@@ -384,6 +385,21 @@ def set_id(self, value):
             raise TypeError('id must be a string, not {0}'.format(type(value)))
         self._id = value
 
+    def heartbeat(self, heartbeat, pipeline=None):
+        self.last_heartbeat = heartbeat
+        connection = pipeline if pipeline is not None else self.connection
+        connection.hset(self.key, 'heartbeat', utcformat(self.last_heartbeat))
+
+    def get_heartbeat(self, refresh=True):
+        if refresh:
+            raw = self.connection.hget(self.key, 'heartbeat')
+            if raw:
+                self.last_heartbeat = str_to_date(raw)
+            else:
+                self.last_heartbeat = None
+
+        return self.last_heartbeat
```

Comment:
I think we can remove this method, it's unnecessary. When we fetch `job` from Redis, you can already call `job.last_heartbeat`. If you want to refresh job metadata from, call `job.refresh()`.